en/02_quicktour.md

```diff
@@ -0,0 +1,276 @@
+
+The two records that was inserted has been returned as expected. You find that the previous `insert` was processed successfully.
+
+Previously in the `select` examples, we obtained our results as `Map[String, Any]`, but let's change it so that the results are mapped to a `Member` class.
+
+ScalikeJDBC doesn't require you to do any special configuration to the class mapped from `ResultSet`. It is OK to simply define it as a `case class` or just as a regular `class`.
+
+It is recommended to define `NOT NULL` columns as `Option` types and use DateTime and LocalDate from [Joda Time](http://www.joda.org/joda-time/) for date and timestamp columns. The Date Time API of Java SE 8 can be used as well, but I will explain it separately. In this sample, I will show you an example of using Joda Time.
+
+    case class Member(
+      id: Long, 
+      name: String, 
+      description: Option[String] = None, 
+      birthday: Option[LocalDate] = None, 
+      createdAt: DateTime)
+    
+    val allColumns = (rs: WrappedResultSet) => Member(
+      id = rs.long("id"), 
+      name = rs.string("name"), 
+      description = rs.stringOpt("description"),
+      birthday = rs.dateOpt("birthday").map(_.toLocalDate), 
+      createdAt = rs.jodaDateTime("created_at")
+    )
+    
+    val members: List[Member] = DB readOnly { implicit session =>
+      SQL("select * from members limit 10").map(allColumns).list.apply()
+    }
+    // => members: List[Member] = List(Member(1,Alice,None,Some(1980-01-01),2012-12-31T00:02:09.247+09:00), Member(2,Bob,None,None,2012-12-31T00:02:09.247+09:00))
+
+### SQL Interpolation
+
+Since [String Interpolation (SIP-11)](http://docs.scala-lang.org/sips/pending/string-interpolation.html) was introduced to Scala 2.10.0, you can embed expressions to strings by `${...}`.
+
+ScalikeJDBC offers an extension called "SQL interpolation" which takes advantage of this feature.
+
+While there is a risk of causing an SQL injection vulnerability in misuse of `SQL("...")`, you don't have a worry of its happening with `sql"..."` because all external inputs become binding variables.
+
+So let's try the SQL interpolation. Instead of writing like this as previously seen;
+
+    def create(name: String, birthday: Option[LocalTime])(implicit session: DBSesion): Member = {
+      val id = SQL("insert into members (name, birthday) values ({name}, {birthday})")
+        .bindByName('name -> name, 'birthday -> birthday)
+        .updateAndReturnGeneratedKey.apply()
+      Member(id, name, birthday)
+    }
+
+    def find(id: Long)(implicit session: DBSesion): Option[Member] = {
+      SQL("select id, name, birthday from members where id = {id}")
+        .bindByName('id -> id)
+        .map { rs => Member(rs.long("id"), rs.string("name"), rs.timestampOpt("birthday").map(_.toDateTime) }
+        .single.apply()
+    }
+
+we can write it as below. Notice that it has also become simpler as passing binding variables by `#bindByName` being no longer there.
```

Comment:
Now we can express the same thing as below. It is much simpler than the above one because we don't need to pass named variables by using `#bindByName` any more.